rpcs3/Emu/Audio/Cubeb/CubebBackend.cpp

```diff
@@ -196,13 +203,14 @@ void CubebBackend::CloseUnlocked()
 	m_last_sample.fill(0);
 
 	m_default_dev_changed = false;
-	m_current_device.clear();
+	m_default_device.clear();
 }
 
 void CubebBackend::Close()
 {
 	std::lock_guard lock(m_cb_mutex);
 	std::lock_guard dev_sw_lock{m_dev_sw_mutex};
+	std::lock_guard state_cb_lock(m_state_cb_mutex);
```

Comment:
Although it's merged I have to say anyway that std::scoped_lock includes optimizations for multiple lockings.